core/server/src/main/java/alluxio/master/AlluxioMaster.java

```diff
@@ -71,12 +71,13 @@
    */
   public static void main(String[] args) {
     if (args.length != 0) {
-      LOG.info("java -cp {} {}", Version.ALLUXIO_JAR, AlluxioMaster.class.getCanonicalName());
+      LOG.info("java -cp {} {}", RuntimeConstants.ALLUXIO_JAR,
+          AlluxioMaster.class.getCanonicalName());
       System.exit(-1);
     }
 
     // validate the conf
-    if (!ValidateConf.validate()) {
+    if (!ValidateConf.validate(Configuration.createServerConf())) {
```

Comment:
I updated the PR so alluxio shell will check the client-side conf before running.
for client conf in general, I didn't check it we don't have a good entrance point to check it (as in master or worker or shell, we have the main to check). It is possible that we could have it in `createClientConf`, but then it will be strange why we check in `createClientConf` but not in `createServerConf`